.github/workflows/e2e.yml

```diff
@@ -0,0 +1,74 @@
+name: Operator e2e tests
+
+on: 
+  push:
+     branches: [main]
+  issue_comment:
+    types: [created]
+  workflow_dispatch: {}
+
+jobs:
+
+  build:
+    
+    if: | 
+     (github.event.issue.pull_request && github.event.comment.body == '/run-e2e') ||
```

Comment:
We should update CONTRIBUTING.md to document how to trigger the e2e tests in your PR.  There is a section for the PR (step 7).  We can probably add a part there for CI, unit tests are run automatically for each commit in the PR and '/run-e2e' to trigger the end-to-end tests.

Reply:
ok will do that.